policy/intrinsic_zernikes/comcam/intrinsic_zk__31_23.yaml

```diff
@@ -0,0 +1,194 @@
+--- 
+         
+ # TODO: This file needs to be updated once batoid allows for its generation
+ # for now this is just the same as comcam/intrinsic_zk_g_31_23.yaml
+ 
+ # Intrinsic Zk for the REF band 500 nm.         
+ # The first dimension is kmax = 31, corresponds to number of Zernike         
```

Comment:
The first dimension is kmax = 31 --> The first dimension has the kmax = 31.

This applies to all the `yaml` files.